### PR TITLE
PROC-957 As a reseller I want to list my organizations

### PR DESCRIPTION
**Pull request by MollieRick**

### CODE REVIEW

File: source/reference/v2/organizations-api/list-organizations.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+.. authentication::
+    :api_keys: true
+    :oauth: false
+
+Retrieve all organizations that are connected your partner-account.
+
+The results are paginated. See :doc:`pagination </guides/pagination>` for more information.
+
+.. note:: This API is only for partners.
```
Comment (ndijkstra) [inline source/reference/v2/organizations-api/list-organizations.rst]: Maybe a nice link to our Partners page :)

File: source/reference/v2/organizations-api/list-organizations.rst [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Example
+-------
+
+Request
+^^^^^^^
+.. code-block:: bash
+   :linenos:
+
+   curl -X GET https://api.mollie.com/v2/organizations?limit=5 \
```
Comment (ndijkstra) [inline source/reference/v2/organizations-api/list-organizations.rst]: I would leave out the `?limit=5`. We don't have that in the other examples as well.

File: source/reference/v2/organizations-api/list-organizations.rst [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+           ]
+       },
+       "count": 5,
+       "_links": {
+            "documentation": {
+                "href": "https://docs.mollie.com/reference/v2/organizations-api/list-organizations",
+                "type": "text/html"
+            },
+            "self": {
+                "href": "https://api.mollie.dev/v2/organizations?limit=5",
```
Comment (ndijkstra) [inline source/reference/v2/organizations-api/list-organizations.rst]: `api.mollie.dev` > `api.mollie.com`

File: source/reference/v2/organizations-api/list-organizations.rst (hunk 2 above)
Comment (MollieRick) [reply source/reference/v2/organizations-api/list-organizations.rst]: It is in the `List Payments` as well, because it can be a long list.

**Comment by willemstuursma** [comment]
I think the Get customer has a missing link to this (text that is not a link).

File: source/reference/v2/organizations-api/list-organizations.rst [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+----------
+.. list-table::
+   :widths: auto
+
+   * - ``from``
+
+       .. type:: string
+          :required: false
+
+     - Offset the result set to the payment with this ID. The payment with this ID is included in the result
```
Comment (willemstuursma) [inline source/reference/v2/organizations-api/list-organizations.rst]: copy pasta

**Comment by MollieRick** [comment]
> I think the Get customer has a missing link to this (text that is not a link).

@willemstuursma Where?

File: source/theme/sidebar-reference-v2.html [hunk 5]
```diff
@@ -337,6 +337,11 @@
 			'Get organization',
 			'GET /v2/organizations/*'
 		) }}
+		{{ sidebar_item(
+		'reference/v2/organizations-api/list-organizations',
+		'List organizations',
```
Comment (willemstuursma) [inline source/theme/sidebar-reference-v2.html]: alignment is off with other sidebar items

File: source/reference/v2/organizations-api/list-organizations.rst [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+==================
+.. api-name:: Organizations API
+    :version: 2
+
+.. endpoint::
+    :method: GET
+    :url: https://api.mollie.com/v2/organizations
+
+.. authentication::
+    :api_keys: true
```
[the remark this answers is not in the document]
Comment (MollieRick) [reply source/reference/v2/organizations-api/list-organizations.rst]: But that is strange, right? Merchants are linked to the partner without an app intervening, so why do we want that?

**Pull request merged**